docs/source/faq.rst

```diff
@@ -795,10 +793,7 @@ And each user block will be followed by a pagebreak, except the last user.
         return angularParser(tag);
     }
     const doc = new Docxtemplater();
```

Comment:
You can now remove the line : 

    const doc = new Docxtemplater();

Reply:
sure.
Done in [this](https://github.com/shadab14meb346/docxtemplater/commit/cc4ff0e55fbaf1cd1c4e3dee9def0bca2913fdd6) commit.